modules/storages/app/components/storages/admin/storage_list_component.sass

```diff
@@ -0,0 +1,27 @@
+.op-storage-list
+  display: grid
+  grid-template-columns: fit-content(180px) 1fr 180px 100px 240px
+  grid-template-areas: "name label user provider time" "host host user provider time"
+
+  .op-principal--name
+    @extend .text-light
+
+  @media screen and (max-width: $breakpoint-md)
+    grid-template-columns: fit-content(180px) 1fr
+    grid-template-areas: "name label" "host host" "user user" "time time"
+    grid-row-gap: 5px
+
+    &--provider
+      display: none
```

Comment:
There is a system argument, which you can set to hide elements on certain breakpoints `hide: :sm`. Same for the user and time element

Reply:
Oh, I didn't know that- nice one 👍🏾